lib/mapper.js

```diff
@@ -100,8 +102,8 @@ function getOrGenerateDevId(facade, payload) {
     if (devId && devId.length > 0) {
         return devId;
     }
-    if (payload.username) {
-        devId = "nodev-segment-" + hash(payload.username);
+    if (payload.user_id) {
+        devId = "nodev-segment-user-id-" + hash(payload.user_id);
     } else if (payload.user_email) {
```

Comment:
Can we add in:
else if(payload.username) {
    devId = "nodev-segment-" + hash(payload.username);
}
It looks like we lost that part when we added in hashing for the user_id used in dev_id. We want to populate a kahuna dev_id with a hash of the user_id, username, or email in that order. Thanks!